src/main/java/org/sonarlint/intellij/config/global/wizard/NotificationsStep.java

```diff
@@ -84,7 +84,8 @@ public Object getPreviousStepId() {
     if (onlyEditNotifications) {
       return null;
     }
-    if (model.getServerType() == WizardModel.ServerType.SONARCLOUD) {
+    if (model.getServerType() == WizardModel.ServerType.SONARCLOUD || CodescanCloudConstants.CODESCAN_US_URL.equals(model.getServerUrl())
```

Comment:
What if user enters URL with slash at the end?
`https://app.codescan.io`

Reply:
It is handled in ServerStep.java.

Reply:
For SonarLintUtils.isCodeScanCloudAlias(String url), we need an exact match of url. Hence, I check for the domain name entered by the user and based on it, set the url as https://app.codescan.io or https://app-eu.codescan.io .
The cases of user using http protocol, trailing slash are addressed by this approach.